docs/content/manual/virtual-machines/clone-vm-and-don't-select-start-after-creation.md

```diff
@@ -1,11 +1,27 @@
 ---
 title: Clone VM and don't select start after creation	
 ---
+
+## Case 1
 1. Clone VM from Virtual Machine list
```

Comment:
qq: How is the root disk allocated in a VM without the volume?

Reply:
So a clone without volume is what was being called a clone template. It's creating a new VM with the metadata from the previous one, but not cloning the longhorn volume. So it will default to the same size and type I believe.